depmap_analysis/network_functions/indra_network.py

```diff
@@ -1570,3 +1581,14 @@ def list_all_hashes(ksp_results):
                     hash_set.update([item['stmt_hash'] for item in
                                      meta_list])
     return list(hash_set)
+
+
+def get_subgraph_from_mesh_ids(indra_graph, mesh_ids):
+    """Builds a subgraph with edges related to supplied mesh_ids"""
+    related_hashes = find_related_hashes(mesh_ids)
+    subgraph = nx.DiGraph
```

Comment:
You need to call the class here:
```suggestion
    subgraph = nx.DiGraph()
```